framework/kms-keyring.md

```diff
@@ -73,6 +73,9 @@ and MAY return a KMS Client that can make the following API calls in the given A
 
 The keyring will use this client supplier to determine the KMS client to use when making KMS calls.
 
+The purpose of the client supplier is
+to control how the keyring communicates with AWS KMS.
```

Comment:
```suggestion
to control how the KMS Keyring communicates with AWS KMS.
```

Reply:
Per guidance from @juneb, "keyring" is not a proper noun, so should not be capitalized.
```suggestion
to control how the KMS keyring communicates with AWS KMS.
```